### DISCUSSION TITLE
Next-Urql problem, probably my own mistake again :D

### DISCUSSION THREAD

**Discussion by BjoernRave** [urql-graphql/urql#1100] [Q&A] [answered]
When updating next-urql and all other urql related packages to the latest version, I get the following error:

```
error - ./node_modules/next-urql/dist/next-urql.es.js 4:8
Module parse failed: Unexpected token (4:8)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| //# sourceMappingURL=next-urql.es.js.map
| 
>         });
|         forceUpdate[1](forceUpdate[0] + 1);
|       };
```

I didnt create an issue, because in the end it's probably just a mistake on my side again, but I already tried to remove the `yarn.lock` and install `node_modules` again

**Comment by kitten** [comment, marked as answer]
Not your own mistake :) We accidentally published a broken build, see: https://github.com/FormidableLabs/urql/issues/1101
If you upgrade to next-urql@2.1.1 you'll find that it has been fixed ✌️

**Reply by BjoernRave** [reply]
ah, that's great news :)
